javascript/ql/src/semmlecode.javascript.dbscheme

```diff
@@ -560,7 +560,7 @@ case @typeexpr.kind of
 | 10 = @parenthesized_typeexpr
 | 11 = @tuple_typeexpr
 | 12 = @keyof_typeexpr
-| 13 = @qualifiedtypeaccess
+| 13 = @qualified_typeaccess
```

Comment:
Now we have `localtype_access` and `qualified_typeaccess` 😕 

I'd suggest `local_type_access` and `qualified_type_access`.